src/protocols/telnet/settings.c

```diff
@@ -63,6 +63,7 @@ const char* GUAC_TELNET_CLIENT_ARGS[] = {
     "wol-send-packet",
     "wol-mac-addr",
     "wol-broadcast-addr",
+    "wol-broadcast-port",
```

Comment:
As with the `broadcast_port` parameter, I suspect that `wol-broadcast-port` may be technically incorrect, and that something like `wol-port` would be preferable.